Extensions/Terraform/Src/Tasks/TerraformTaskV1/task.json

```diff
@@ -0,0 +1,318 @@
+                "aws": "aws",
+                "gcp": "gcp"
+            },
+            "properties": {
+                "EditableOptions": "False"
+            }
+        },
+        {
+            "name": "command",
+            "type": "pickList",
+            "label": "Command",
+            "defaultValue": "init",
+            "required": true,
+            "helpMarkDown": "The terraform command to execute.",
+            "options": {
+                "init": "init",
+                "validate": "validate",
+                "plan": "plan",
+                "apply": "apply",
+                "destroy": "destroy"
+            },
+            "properties": {
+                "EditableOptions": "False"
+            }
+        },
+        {
+            "name": "workingDirectory",
+            "type": "filePath",
+            "label": "Configuration Directory",
+            "defaultValue": "$(System.DefaultWorkingDirectory)",
+            "helpMarkDown": "Directory containing the Terraform configuration files. The default value is $(System.DefaultWorkingDirectory)",
+            "required": "false"
+        },
+        {
+            "name": "commandOptions",
+            "type": "string",
+            "label": "Additional Command Arguments",
+            "helpMarkDown": "Any additional arguments for the selected command such as '-option=value'. Multiple options can also be provided delimited by spaces.",
+            "required": false
+        },
+        {
+            "name": "environmentServiceNameAzureRM",
+            "type": "connectedService:AzureRM",
+            "label": "Azure Environment Subscription",
+            "required": false,
+            "visibleRule": "provider = azurerm && command != init && command != validate",
+            "helpMarkDown": "Select an Azure Resource Manager subscription for the deployment"
+        },
+        {
+            "name": "environmentServiceNameAWS",
+            "type": "connectedService:AWS",
+            "label": "AWS Environment Subscription",
+            "required": false,
+            "visibleRule": "provider = aws && command != init && command != validate",
+            "helpMarkDown": "Select an AWS subscription for the deployment"
+        },
+        {
+            "name": "environmentServiceNameGCP",
+            "type": "connectedService:GoogleCloudServiceEndpoint",
+            "label": "GCP Environment Subscription",
+            "required": false,
+            "visibleRule": "provider = gcp && command != init && command != validate",
+            "helpMarkDown": "Select a GCP subscription for the deployment"
+        },
+        {
+            "name": "backendTypeAzureRM",
+            "type": "pickList",
+            "label": "Backend Type",
+            "defaultValue": "local",
+            "required": false,
+            "helpMarkDown": "The teraform backend type for AzureRM to use for maintaining the state.",
+            "options": {
+                "local": "local",
```

Comment:
Can you make the pick list options more friendly like "Local Disk" and "Azure blob storage"